core.py

```diff
@@ -245,7 +261,7 @@ def cli():
     ),
 )
 @click.pass_context
-def validate(
+def validate(  # noqa: C901
```

Comment:
you should rarely be adding #noqa. We are intentionally using flake8 and its configuration.
Instead, fix the offending issue.